packages/api-explorer/package.json

```diff
@@ -11,49 +27,38 @@
     "@readme/oas-form": "^8.1.0",
     "@readme/oas-to-har": "^8.1.0",
     "@readme/oas-to-snippet": "^8.1.0",
-    "@readme/syntax-highlighter": "^10.0.0",
-    "@readme/ui": "^1.10.2",
+    "@readme/syntax-highlighter": "^10.1.0",
     "@readme/variable": "^8.1.0",
     "classnames": "^2.2.5",
     "fetch-har": "^4.0.2",
     "js-cookie": "^2.1.4",
     "lodash.kebabcase": "^4.1.1",
     "oas": "^5.0.0",
     "prop-types": "^15.7.2",
-    "react": "^16.13.1",
     "react-copy-to-clipboard": "^5.0.1",
     "react-debounce-input": "^3.2.0",
-    "react-dom": "^16.13.1",
     "react-json-view": "^1.19.1",
     "react-waypoint": "^9.0.2",
     "whatwg-fetch": "^3.0.0"
   },
-  "scripts": {
-    "build": "webpack --config webpack.prod.js",
-    "inspect": "jsinspect",
-    "lint": "eslint . --ext js --ext jsx",
-    "pretest": "npm run lint && npm run prettier && npm run inspect",
-    "prettier": "prettier --list-different --write \"./**/**.{js,jsx}\"",
-    "test": "jest --coverage --runInBand",
-    "watch": "webpack -w --progress"
-  },
-  "publishConfig": {
-    "registry": "http://registry.npmjs.org",
-    "access": "public"
+  "peerDependencies": {
+    "@readme/ui": "^1.10.2",
+    "react": "16.x",
+    "react-dom": "16.x"
   },
-  "author": "Dom Harrington <dom@harrington-mail.com>",
-  "license": "ISC",
-  "repository": "https://github.com/readmeio/api-explorer/tree/master/packages/api-explorer",
   "devDependencies": {
     "@readme/eslint-config": "^3.2.0",
     "@readme/oas-examples": "^3.6.0",
+    "@readme/ui": "^1.10.2",
     "@testing-library/dom": "^7.26.3",
     "css-loader": "^4.3.0",
     "eslint": "^7.0.0",
     "jest": "^26.0.1",
     "jsinspect": "^0.12.6",
     "node-fetch": "^2.0.0-alpha.9",
     "prettier": "^2.0.1",
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0",
```

Comment:
Do we need these in dev deps just so they get installed when we `npm i`? If so, soon this won't be necessary (once we manage to bump up to [NPM 7](https://github.blog/2020-10-13-presenting-v7-0-0-of-the-npm-cli/)!)

Reply:
@rafegoldberg yeah. once we're on npm 7 and they're defined as peerdeps we can remove them as being devdeps